src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+
+import { Button } from './Button'
+
+const figmaUrl = 'https://www.figma.com/file/0ar6alIEDe8iYCb8kU7Rxd/%5BDS%5D-Console-Mia-Platform?node-id=2%3A162&mode=dev'
```

Comment:
is this figma public? If not why put it here if no-one can access it?

Reply:
This is how users see it without permissions, on login you are redirected again on this page

![image](https://github.com/mia-platform/design-system/assets/9254840/795c4bf4-51f5-4e3b-8d9b-41c5fec28eb2)

While without adding the figma file would appear like this:

![image](https://github.com/mia-platform/design-system/assets/9254840/b372bd07-6f1a-4e38-9a7d-4873a4f582bc)

Reply:
Figma URL removed ✅